src/harness/testcases/grant_testcase.py

```diff
@@ -624,3 +624,245 @@ def test_WINNF_FT_S_GRA_28(self):
     self.assertEqual(response[1]['cbsdId'], cbsd_id)
     self.assertTrue(response[0]['response']['responseCode'] == 401
                     or response[1]['response']['responseCode'] == 401)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_GRA_2(self):
+    """Successful CBSD grant request.
+    Incumbent is present in the GAA frequency range requested by the
+    CBSD which is outside the protection zone..
+    """
+    # Register the device outside of PPA
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_d.json')))
```

Comment:
I think one-step registration or multi-step registration are possible for both Category (A and B)
like in  the tests:  WINNF.FS.S.GRA.1 OR WINNF.FS.S.GRA.22